### PR TITLE
Expose member info needed for offer page

### PR DESCRIPTION
**Pull request by hedvigoscar**
- New root query: member
- Added postal number to insurance

### CODE REVIEW

File: src/features/insurance/index.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ const loadInsurance = async (
     presaleInformationUrl: insuranceObject.presaleInformationUrl,
     monthlyCost: insuranceObject.currentTotalPrice,
     address: insuranceObject.addressStreet,
+    postalNumber: insuranceObject.zipCode,
```
Comment (palmfjord) [inline src/features/insurance/index.ts]: can we decide on `postalCode`, `postalNumber` or `zipCode` maybe 😅 i'm starting to get confused here

File: src/resolvers/member.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { getUser } from '../api'
+import { QueryToMemberResolver } from '../typings/generated-graphql-types'
+
+const member: QueryToMemberResolver = async (
+  _parent,
+  _args,
+  { getToken, headers },
+) => {
+  const token = await getToken()
+  const memberObj = await getUser(token, headers)
```
Comment (palmfjord) [inline src/resolvers/member.ts]: `member`? we know it's an object already right?

File: src/features/insurance/index.ts (hunk 1 above)
Comment (hedvigoscar) [reply src/features/insurance/index.ts]: I'm trying to standardize on `postalNumber`, but we're exposing `zipCode` from backend in a lot of places.

File: src/resolvers/member.ts (hunk 2 above)
Comment (hedvigoscar) [reply src/resolvers/member.ts]: `member` shadows the function name 😞 I know it's ugly and I'm open to better names

File: src/features/insurance/index.ts (hunk 1 above)
Comment (palmfjord) [reply src/features/insurance/index.ts]: ok, coolio! carry on 👜

File: src/resolvers/member.ts (hunk 2 above)
Comment (palmfjord) [reply src/resolvers/member.ts]: ): `memberResponse`?

**Pull request merged**